src/dotnet/APIView/APIViewWeb/Languages/CppLanguageService.cs

```diff
@@ -39,7 +39,7 @@ public class CppLanguageService : LanguageService
         private const string AccessModifierProtected = "protected";
         private const string AccessModifierPublic = "public";
         private const string RootNamespace = "Azure";
-        private const string DetailsNamespacePostfix = "::Details";
```

Comment:
Is "Details" here still in use anywhere? Just want to make sure it is not breaking anything

Reply:
No, this is an internal namespace only.  It won't break anything.